browser/search_engines/search_engine_provider_util.cc

```diff
@@ -34,4 +35,16 @@ bool IsRegionForQwant(Profile* profile) {
       TemplateURLPrepopulateData::PREPOPULATED_ENGINE_ID_QWANT;
 }
 
+bool IsOTRGuestProfile(Profile* profile) {
```

Comment:
I'm still confused by this. Is there more than one kind of guest profile?

Reply:
I guess it depends on what you call a guest profile. IsGuestSession is set to true for regular and off-the-record profiles of a profile with "guest" path. GetProfileType only returns GUEST_PROFILE type for the off-the-record profile.

Reply:
For a short while (5/20 - 5/29) Chromium had an ideal method Profile::IsGuestProfile for what we need, but they removed it: https://chromium.googlesource.com/chromium/src/+/5f95c46d5ce37d3b271b1f99063459e5d715761e